Makefile

```diff
@@ -74,7 +85,7 @@ ifneq ($(HAS_WEBAPP),)
 	cd webapp && $(NPM) run build;
 endif
 
-## Builds the webapp in debug mode, if it exists.
+## Builds the webapp, in debug mode, if it exists.
 .PHONY: webapp-debug
```

Comment:
Would you be willing to move over the webapp-debug logic from incident response also? I find it useful almost everyday when working on the frontend.

Reply:
Yes! I realized after digging in a bit that I had missed part of your improvements. This is also a good note to myself to update README.md with something to help guide developers starting with this template.

Reply:
Moved! I reworked it slightly to use `pluginctl` to redeploy (with a quick `bundle` vs. having to rebuild everything) so that we don't have to assume `mattermost-server` is a peer directory at all. It's now just `make watch` to get this functionality.